app/main.py

```diff
@@ -1 +1,40 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (
+            f"{{Name: {self.name}, "
+            f"Health: {self.health}, "
+            f"Hidden: {self.hidden}}}"
+        )
+
+    @classmethod
+    def update_alive(cls) -> None:
+        cls.alive = [animal for animal in cls.alive if animal.health > 0]
```

Comment:
Don't u think that it is overkill to iterate through all instances just to remove 1? I think it is better to remove it in `bite` function

Reply:
agree, fixed